harmony_browse_image_generator/color_utility.py

```diff
@@ -7,6 +7,8 @@
 
 import requests
 from harmony.message import Source as HarmonySource
+import numpy as np
+from numpy import ndarray
```

Comment:
Yup. I'm surprised I can't have ndarray in a TYPE_CHECKING guard, because that's all it's used for.  I can change them all to np.ndarray, but I remember the format got black wonky.  

Ok, I tried again.  I can update it by putting my type in single quotes. Is that better?

``` python
from typing import TYPE_CHECKING
if TYPE_CHECKING:
    from numpy import ndarray

```

And then 

``` python
def remove_alpha(raster: 'ndarray') -> tuple['ndarray', 'ndarray', None]:
    """remove alpha layer when it exists."""
    if raster.shape[0] == 4:
        return raster[0:3, :, :], raster[3, :, :]
    return raster, None
```

Reply:
3dbba3c

Reply:
Is that even right?

Reply:
So, I'm worried this next comment is me missing something, but couldn't you just stick with the single import (`import numpy as np`) and then:

```
def remove_alpha(raster: np.ndarray) -> tuple[np.ndarray, np.ndarray, None]:
```

I tried a toy model locally, and that seemed okay.

Reply:
I could. This was why I hadn't elsewhere: "but I remember the format got black wonky"

Reply:
but I don't have strong opinions I suppose 9a7902f